src/server.js

```diff
@@ -1,26 +1,62 @@
 const express = require('express');
 const morgan = require('morgan');
+const bodyParser = require('body-parser');
 
 const { SERVER_PORT } = require('./config');
 
 const PG = require('./pg');
+const NEO = require('./neo');
+
+const currentDb = PG;
 
 const server = express();
 
-async function findAll(type, options) {
+server.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+async function findAll(req, type, options, responseDb = currentDb) {
```

Comment:
It might be nice to pass the DB as the first argument, since it's basically the most important one.